lib/sky-pages-module-generator.js

```diff
@@ -29,7 +36,7 @@ function getSource(skyAppConfig) {
       if (/^http/.test(filePath)) {
         hasExternal = true;
       } else {
-        styles += `require('style-loader!${filePath}');\n`;
+        styles += `require('!style-loader!css-loader!sass-loader!${filePath}');\n`;
```

Comment:
Using `css-loader` so that `@import` statements in CSS files are resolved (the latest version of Angular was having trouble resolving `@import` statements in CSS files that were being injected via `require()`.